common/statistics/src/report.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+use crate::clients::{
+    gateway_conn_statistics::GatewayStats, nym_api_statistics::NymApiStats,
+    packet_statistics::PacketStatistics,
+};
+
+use super::error::StatsError;
+
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+/// Report object containing both data to be reported and client / device context. We take extra care not to overcapture context information.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct ClientStatsReport {
+    pub(crate) last_update_time: OffsetDateTime,
+    pub(crate) client_id: String,
+    pub(crate) client_type: String,
+    pub(crate) os_information: os_info::Info,
```

Comment:
be extremely careful with it. as we found out, some operator, and I'd also assume users, do not want to report their OS information. if possible make it togglable with config (like we have in `nym-node`)

Reply:
So, for now there is no collection anyway, as there is no service provider set up, so no address gets distributed. 
Once there will be, it will probably be up to the SDK using code to choose wether to report stuff or not